lib/stdlib/src/shell_docs_test.erl

```diff
@@ -0,0 +1,113 @@
+%%
+%% %CopyrightBegin%
+%%
+%% Copyright Ericsson AB 1996-2024. All Rights Reserved.
+%%
+%% Licensed under the Apache License, Version 2.0 (the "License");
+%% you may not use this file except in compliance with the License.
+%% You may obtain a copy of the License at
+%%
+%%     http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing, software
+%% distributed under the License is distributed on an "AS IS" BASIS,
+%% WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+%% See the License for the specific language governing permissions and
+%% limitations under the License.
+%%
+%% %CopyrightEnd%
+%%
+-module(shell_docs_test).
+-moduledoc false.
+
+-include_lib("kernel/include/eep48.hrl").
+
+-export([module/2]).
+
+module(#docs_v1{ docs = Docs, module_doc = MD }, Bindings) ->
+    MDRes = parse_and_run(module_doc, MD, Bindings),
+    case lists:flatten(
+           [parse_and_run(KFA, EntryDocs, Bindings)
+            || {KFA, _Anno, _Sig, EntryDocs, _Meta} <- Docs, is_map(EntryDocs)] ++ MDRes) of
+        [] ->
+            ok;
+        Else ->
+            Else
+    end.
+
+parse_and_run(KFA, #{} = Ds, Bindings) ->
+    [parse_and_run(KFA, D, Bindings) || _ := D <- Ds];
+parse_and_run(KFA, Docs, Bindings) ->
+    InitialBindings = proplists:get_value(KFA, Bindings, erl_eval:new_bindings()),
+    io:format("Testing: ~p~n",[KFA]),
+    case test(inspect(shell_docs_markdown:parse_md(Docs)), InitialBindings) of
+        [] -> [];
+        Else ->
+            {KFA, lists:flatten(Else)}
+    end.
+
+test({pre,[],[{code,Attrs,[<<">",_/binary>> = Code]}]}, Bindings) ->
+    case proplists:get_value(class, Attrs, ~"language-erlang") of
+        ~"language-erlang" ->
+            run_test(Code, Bindings);
+        _ ->
+            test(Code, Bindings)
+    end;
+test({pre,[],[{code,[{class,~"language-erlang"}],[<<">",_/binary>> = Code]}]}, Bindings) ->
+    run_test(Code, Bindings);
```

Comment:
with the change above, I think this case is contained in the case above :)

Reply:
Yeah, this is covered by the above. Will delete.